src/components/common/@Icons/loginIcon.tsx

```diff
@@ -0,0 +1,35 @@
+const LoginIcon = () => {
+  return (
+    <svg width="52" height="52" viewBox="0 0 52 52" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        fillRule="evenodd"
```

Comment:
나중에 onClick 이벤트를 주려면 onClick Props 를 받아야합니다
또한 사이즈 변경이 용이하게 width, height 도 받아오면 좋을 것 같아요~ 

```suggestion
type SVGIconProps = {
width?:number,
height?:number,
onClick?:()=>void
}
const LoginIcon = ( {width=52, height=52, onClick : SVGIconProps ) => {
  return (
    <svg width={width} height={height} viewBox="0 0 52 52" fill="none"  onClick={onClick ? onClick : () => {}} xmlns="http://www.w3.org/2000/svg">
      <path
        fillRule="evenodd"
```

Reply:
말씀 해주신 내용대로 반영하겠습니다 😀!!